app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,12 @@
 class ApplicationMailer < ActionMailer::Base
   default from: Settings.mail.from
   layout 'mailer'
+
+  def dwp_is_down_notifier
+    to = Settings.mail.dwp_notification_alert
+    mail(to: to, subject: t('email.dwp_alert_notification..subject')) do |format|
+      format.html { render plain: t('email.dwp_alert_notification.message') }
```

Comment:
is this ok ? rendering plain text in html email ?

Reply:
@garytaylor I was thinking about it but I couldn't find any reason why this should be an issue.

Reply:
do we need the html format?

Reply:
@leoapost maybe we don't it theory it should be fine to render text only.